plugins/catalog-backend/src/database/CommonDatabase.ts

```diff
@@ -283,22 +283,48 @@ export class CommonDatabase implements Database {
 
     let builder = tx<DbEntitiesRow>('entities');
     for (const [index, filter] of (filters ?? []).entries()) {
+      const key = filter.key.replace('*', '%');
```

Comment:
Do we really need to support wildcards in keys? 😅

Reply:
It was trivial when the structure was in there, and I thought there were some actual use cases, so I just went for it :)